packages/@react-spectrum/button/docs/Button.mdx

```diff
@@ -161,3 +162,14 @@ function Example() {
 ```tsx example
 <Button variant="cta" isDisabled>Save</Button>
 ```
+
+### Icon only
+[View guidelines](https://spectrum.adobe.com/page/button/#Label-and-icon)
+
+```tsx example
+<Flex direction="row" gap={8}>
+  <Button variant="cta"><Bell /></Button>
```

Comment:
The buttons need `aria-label` props.

Reply:
We should also provide examples of buttons with both an icon and a label, where the icon has meaning that is separate from the label.

Reply:
For example:

```html
<Button variant="primary">
  <Add  aria-label="Add" />
  <Text>Event</Text>
</Button>
```

or

```html
<Button variant="primary">
  <Text>Add</Text>
  <Comment  aria-label="Comment" />
</Button>

<Button variant="primary">
  <Text>Remove</Text>
  <Comment  aria-label="Comment" />
</Button>
```